tests/ServerRequestTest.php

```diff
@@ -340,7 +340,7 @@ public function dataGetUriFromGlobals()
                 array_merge($server, ['REQUEST_URI' => '/blog/article.php']),
             ],
             'Empty server variable' => [
-                'http://localhost',
+                'http:',
```

Comment:
Not sure about this line, but I did not want to revert this commit: f3d63974d67c15a7

Reply:
Is this test case still required?